### PR TITLE
Hw 03

### PR DESCRIPTION
**Pull request by oleg-gerasimovich**
Please, check my 3-d homework

### CODE REVIEW

File: homework/hw_03/task1.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+using System;
+
+namespace CSPLess3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+
```
Comment (VadzimPapko) [inline homework/hw_03/task1.cs]: remove unused empty lines

File: homework/hw_03/task1.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            object @boolean = r;
+            byte w = Convert.ToByte((bool)@boolean);
+
+            //3 пример - упаковка char и распаковка в int
+            char e = 'F';
+            object @char = e;
+            int unbox = (int)(char)@char;
+
+
+
```
Comment (VadzimPapko) [inline homework/hw_03/task1.cs]: do not put empty lines here

File: homework/hw_03/task2.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+namespace CSPLess3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+
+            
+
```
Comment (VadzimPapko) [inline homework/hw_03/task2.cs]: ???

File: homework/hw_03/task2.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            int a1 = 23;
+            Console.WriteLine(a1.GetType());
+            Int32 a2 = 32;
+            Console.WriteLine(a2.GetType());
+
+
+
+
+
```
Comment (VadzimPapko) [inline homework/hw_03/task2.cs]: ??

File: homework/hw_03/task2.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        static void Main(string[] args)
+        {
+
+
+            
+
+            int a1 = 23;
+            Console.WriteLine(a1.GetType());
+            Int32 a2 = 32;
+            Console.WriteLine(a2.GetType());
```
Comment (VadzimPapko) [inline homework/hw_03/task2.cs]: you also have to put the same logic to others simple types, for instance, double, bool, char, etc.

**Comment by oleg-gerasimovich** [comment]
ok, thanks

  
>Воскресенье, 15 марта 2020, 17:22 +03:00 от Vadzim Papko <notifications@github.com>:
> 
> 
>@VadzimPapko approved this pull request.
>Done. Second task is not completed fully. Your score is 7.
>----------------------------------------------------------------------
>In  homework/hw_03/task1.cs :
>> +
>+
>
>remove unused empty lines
>----------------------------------------------------------------------
>In  homework/hw_03/task1.cs :
>> +
>+
>+
>
>do not put empty lines here
>----------------------------------------------------------------------
>In  homework/hw_03/task2.cs :
>> +
>+
>+            
>+
>
>???
>----------------------------------------------------------------------
>In  homework/hw_03/task2.cs :
>> +
>+
>+
>+
>+
>
>??
>----------------------------------------------------------------------
>In  homework/hw_03/task2.cs :
>> +            int a1 = 23;
>+            Console.WriteLine(a1.GetType());
>+            Int32 a2 = 32;
>+            Console.WriteLine(a2.GetType());
>
>you also have to put the same logic to others simple types, for instance, double, bool, char, etc.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
С уважением,
oleg Gerasimovich
oleg-younost@mail.ru

**Pull request merged**